datacube/testutils/io.py

```diff
@@ -2,19 +2,24 @@
 #
 # Copyright (c) 2015-2025 ODC Contributors
 # SPDX-License-Identifier: Apache-2.0
+import inspect
+from collections.abc import Callable, Sequence
 from pathlib import Path
 from types import SimpleNamespace
 from typing import Any
 
 import numpy as np
 import toolz
-import xarray
+import xarray as xr
 from odc.geo import wh_
 from odc.geo.geobox import GeoBox, zoom_to
+from odc.geo.geobox import pad as gbox_pad
 from odc.geo.warp import resampling_s2rio
 from odc.geo.xr import xr_coords
 from typing_extensions import override
 
+from datacube import Datacube
```

Comment:
Would be better to import this relatively, as per the imports below. e.g.

`from ..api import Datacube`